src/components/ImportRecordsFlow/stepThree/assignFieldModal.js

```diff
@@ -82,9 +77,8 @@ export default function AssignFieldModal(props) {
                 />
             }
         >
-            <div className={getModalContainerClassNames()}>
-                <Select
-                    className="rainbow-import-records-flow_step-three-field-select"
+            <StyledModalContainer fileFieldsToAssign={!fileFieldsToAssign.length}>
```

Comment:
instead of pass fileFieldsToAssign let's create:
```const  hasNotFileFieldsToAssign = !fileFieldsToAssign.length;```
and then:
```<StyledModalContainer hasNotFileFieldsToAssign={hasNotFileFieldsToAssign}>```